packages/dinit/build.sh

```diff
@@ -0,0 +1,56 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/davmac314/dinit
+TERMUX_PKG_DESCRIPTION='Service monitoring / "init" system'
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.19.4
+TERMUX_PKG_SRCURL=git+https://github.com/davmac314/dinit.git
+TERMUX_PKG_GIT_BRANCH="r$TERMUX_PKG_VERSION"
```

Comment:
Since there are release tarballs available it would be preferable if we can use those.
```suggestion
TERMUX_PKG_SRCURL=https://github.com/davmac314/dinit/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
TERMUX_PKG_SHA256=3c0f624eb958f8e884631be4ef687da1e475ebaa6241e7ee330b864e6cd9e30b
```

Unless there's a specific reason we need a git checkout.

Reply:
Using a git checkout can avoid backdoor in release tarballs (like XZ utils).

Reply:
That's not even slightly how that incident worked...

In the XZ situation the project release tarballs were tampered with.
The generic `archive/refs/tag/${TAGNAME}.tar.gz` tarballs for GitHub repos ***are*** a git checkout of that tag, without the commit history, which is usually irrelevant for building a project.

Using a `git+` source URL for "security reasons" just doesn't hold up as an explanation.